authui/src/authflowv2/password-policy.ts

```diff
@@ -108,13 +107,14 @@ function checkPasswordSymbol(value: string, el: HTMLElement) {
   };
 }
 
-function checkPasswordStrength(
+async function checkPasswordStrength(
   value: string,
   el: HTMLElement,
   currentMeter: HTMLElement
 ) {
   let isViolated = false;
   const minLevel = Number(el.getAttribute("data-min-level"));
+  const { default: zxcvbn } = await import("zxcvbn");
```

Comment:
Maybe we should load zxcvbn once the controller is connected?

Reply:
connect already calls `this.check()` once, you prefer moving it there explicitly?

Reply:
Oh I think it is ok too.

Reply:
Sorry I deleted the comment because the check is already exited when value is empty, maybe I still add `await import("zxcvbn");` in `connect` to try downloading the bundle earlier?

Reply:
Amended in cbda0cb, not awaited since dynamic import de-duplicates bundle download